### PR TITLE
Ab gallery

### PR DESCRIPTION
**Pull request by djbrownbear**
fixes to gallery - image size, smoother hover transitions

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -2853,6 +2862,10 @@ button:disabled,
 #menu ul a.opener.active:before, #menu ul span.opener.active:before {
   transform: rotate(-180deg);
 }
+#menu ul #navbar-selected {
```
Comment (djbrownbear) [inline assets/css/main.css:2927]: I noticed the menu highlights did not carry over the first time I pushed, so I added the changes to the _menu.scss file.

File: assets/sass/components/_image.scss [hunk 2]
```diff
@@ -45,8 +45,20 @@
 
 			img {
 				width: 100%;
+				height: 200px;
+				object-fit: cover;
+				object-position: center;
+			
+				@include breakpoint(medium) {
```
Comment (djbrownbear) [inline assets/sass/components/_image.scss:52]: there may be a cleaner way to do this, but I wanted the images to be a bit more responsive. on larger screens a lot of the cake image was cut off.

File: assets/css/main.css (hunk 1 above)
Comment (Yu-HuanWu) [reply assets/css/main.css:2927]: it looks like you may have created a branch from a main branch that's old. Do you know if that's what may have happened?

File: assets/css/main.css [hunk 3]
```diff
@@ -1675,9 +1675,25 @@ input[type=radio] + label:before {
 }
 .image.fit img {
   width: 100%;
+  height: 200px;
```
Comment (Yu-HuanWu) [inline assets/css/main.css]: why is height static but width is not? is it because you want it to be rectangle rather than square in most screens?

**Review by Yu-HuanWu — COMMENTED** [review]
q about img ratio

File: assets/css/main.css (hunk 1 above)
Comment (djbrownbear) [reply assets/css/main.css:2927]: I think that may be the case. Confirming I merged in the latest changes from the main branch. Next time I can start a new branch to keep things a bit cleaner too.

File: assets/css/main.css (hunk 3 above)
Comment (djbrownbear) [reply assets/css/main.css]: I updated the heights to instead have a max-height but calculate height based on the calculated width. I like the square look now

File: assets/css/main.css [hunk 4]
```diff
@@ -1675,9 +1675,36 @@ input[type=radio] + label:before {
 }
 .image.fit img {
   width: 100%;
+  height: calc(width * 1.778);
```
Comment (Yu-HuanWu) [inline assets/css/main.css]: why are we multiplying width by 1.778 if we want it to equal to width?

**Comment by djbrownbear** [comment]
I realized I could make things a bit easier by just using css-grid. 
I was able to get the image heights the same and made the gallery a bit more responsive.

**Review by Yu-HuanWu — APPROVED** [review]
lgtm

**Pull request merged**